packages/headless/src/features/configuration/configuration-slice.ts

```diff
@@ -2,17 +2,22 @@ import {isNullOrUndefined} from '@coveo/bueno';
 //@ts-expect-error package is just an alias resolved in esbuild
 import getMagicCookie from '@coveo/pendragon';
 import {createReducer} from '@reduxjs/toolkit';
-import {getOrganizationEndpoints} from '../../api/platform-client';
 import {
-  matchCoveoOrganizationEndpointUrl,
-  isCoveoPlatformURL,
-} from '../../utils/url-utils';
+  getDefaultAnalyticsNextEndpointBaseUrl,
+  getDefaultCommerceEndpointBaseUrl,
+  getDefaultOrganizationEndpointBaseUrl,
+  getDefaultSearchEndpointBaseUrl,
+} from '../../api/platform-client';
+import {matchCoveoOrganizationEndpointUrlAnyOrganization} from '../../utils/url-utils';
 import {
   updateBasicConfiguration as updateCommerceBasicConfiguration,
   updateAnalyticsConfiguration as updateCommerceAnalyticsConfiguration,
   disableAnalytics as disableCommerceAnalytics,
   enableAnalytics as enableCommerceAnalytics,
+  updateProxyBaseUrl as updateCommerceProxyBaseUrl,
+  UpdateProxyBaseUrlPayload,
 } from '../commerce/configuration/configuration-actions';
```

Comment:
I'm kinda curious now as to why the commerce specific stuff isn't kept separate from the base configuration-slice?